packages/block-library/src/template-part/index.php

```diff
@@ -55,7 +55,10 @@ function render_block_core_template_part( $attributes ) {
 			// render the corresponding file content.
 			$template_part_file_path = get_theme_file_path( '/block-template-parts/' . $attributes['slug'] . '.html' );
 			if ( 0 === validate_file( $attributes['slug'] ) && file_exists( $template_part_file_path ) ) {
-				$content = _inject_theme_attribute_in_block_template_content( file_get_contents( $template_part_file_path ) );
+				$content = file_get_contents( $template_part_file_path );
+				$content = is_string( $content ) && '' !== $content
+						? _inject_theme_attribute_in_block_template_content( $content )
+						: '';
```

Comment:
`file_get_contents` can return a string, or a false value. With that in mind, this part of the code can be simplified:
```suggestion
				$content = ( $content )
						? _inject_theme_attribute_in_block_template_content( $content )
						: '';
```

Reply:
Actually they are both valid, and the original - though longer - is more readable so ignore this comment 👍